tests/gold_tests/pluginTest/regex_revalidate/regex_revalidate_state.test.py

```diff
@@ -0,0 +1,132 @@
+'''
+'''
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import os
+import time
+Test.Summary = '''
+regex_revalidate plugin test, MISS (refetch) functionality
+'''
+
+# Test description:
+# If MISS tag encountered, should load rule as refetch instead of IMS.
+# If rule switched from MISS to IMS or vice versa, rule should reset.
+
+Test.SkipUnless(
+    Condition.PluginExists('regex_revalidate.so'),
+    Condition.PluginExists('xdebug.so')
```

Comment:
Why does xdebug have to exist?

Non-experimental plugins should always be present so I don't think checking for regex_revalidate.so is good, test should simply be allowed to fail if it's not presesnt.

Reply:
xdebug not needed for this test.